core/db/entities.py

```diff
@@ -2,7 +2,13 @@
 
 import sqlalchemy as sqla
 
-from core.const import GeographyIndicatorEnum, StateEnum
+# isort: off
+from core.const import (
+    FundingSourceCategoryEnum,
+    GeographyIndicatorEnum,
+    PRAEnum,
+    StateEnum,
+)
```

Comment:
should we use `from core import const as enums` here to prevent lots of imports?

Reply:
I've just imported it as const, as it can potentially contain more than just enums (and "enums" is quite simialar to enum built-in)